dev/parm/config/gfs/config.atmanl.j2

```diff
@@ -17,7 +17,7 @@ if [[ ${DOHYBVAR} = "YES" ]]; then
     export CASE_ANL=${CASE_ENS}
     export BERROR_YAML="atmosphere_background_error_hybrid_${STATICB_TYPE}_${LOCALIZATION_TYPE}"
 else
-    export CASE_ANL=${CASE}
+    export CASE_ANL=${CASE_HIST}
```

Comment:
I am not 100% on this, but does the cubed-sphere output (used in FV3JEDI) also get controlled by this? 
The `LONB_IMO`, `LATB_JMO` are, IIRC, Gaussian points.